### DISCUSSION TITLE
Responsive customer journey map

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/rapid-response-team#57] [Ideas]
Customer journey maps tends to be visually complex, and while they can be pretty, they easily become forgotten because they're not easily accessible to either view or edit.

What if we quickly create and edit a CJM that had its own URL and everything could see and contribute to across our myriad mobile devices?

In [this article for example](https://www.toptal.com/designers/product-design/customer-journey-maps), I'm unlikely to dig much deeper into these because I can't see them, even on my laptop.

**Comment by demi-johnson** [comment]
[This article](https://insidegovuk.blog.gov.uk/2021/04/28/scoping-the-starting-and-sustaining-a-business-whole-user-journey/) talks about a project in the UK where they mapped out a user journey for someone starting and sustaining a business. Starting a business requires contact with various gov agencies and can be a complex process. 

I like how they mapped out the 5 stages of the user journey to show that there's a lot of room for error and the experience as a whole can be improved. Maybe this can tie into the customer journey map idea? Just spit ballin' here!

**Reply by pglevy** [reply]
Spit balling is good! I like the direction they're heading in with the guide. Got me thinking about the [maturity model](https://designsystem.digital.gov/maturity-model/) for USWDS and something like a _path to token adoption for designers_. Could be a simple microsite with steps, tips, resources... 🤔 (cc @bnaolu)
